### PR TITLE
wg-serving: add links to sponsored subprojects

### PR DESCRIPTION
**Pull request by terrytangyuan**
Just added these in custom content in the markdown file directly for now for easy access.

### CODE REVIEW

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

File: wg-serving/README.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ The WG Serving will operate in several workstreams with different areas of focus
 - [Orchestration](https://docs.google.com/document/d/1hbEx3ZEqdXCqWH9RL3uy9FIy35B8pFJ5KiK3HsOz2FE/edit?usp=sharing)
 - [DRA](https://github.com/kubernetes/community/tree/master/wg-device-management)
 
+## Sponsored Subprojects
+
+* [LLM Instance Gateway](https://github.com/kubernetes-sigs/llm-instance-gateway)
```
Comment (BenTheElder) [inline wg-serving/README.md:56]: Subprojects are supposed to be sponsored by SIGs.

WG are supposed to be problem scoped places to discuss across SIGs. They might be involved in projects but can't be the primary owner.

Comment (BenTheElder) [reply wg-serving/README.md:56]: The llm instance gateway repo seems to have the same issue currently, listing the WG as the sponsor. Commented back on the kubernetes/org request for the repo where it seems to be either SIG network or SIG apps

Comment (terrytangyuan) [reply wg-serving/README.md:56]: Thanks for the feedback! This is indeed confusing. Maybe we can rephrase it here instead of "sponsored". Any suggestions?

**Pull request merged**